packages/hooks/feed/useFeedPosting.ts

```diff
@@ -133,30 +130,24 @@ export const useFeedPosting = (
         }
       }
     },
-    [
-      canPayForPost,
-      category,
-      isDAO,
-      makeProposal,
-      mutateAsync,
-      network,
-      postFee,
-      userAddress,
-    ],
+    [category, isDAO, makeProposal, mutateAsync, network, postFee, userAddress],
   );
 
+  const { makeSimplePost, makeVideo, makeTrack, makeArticle, isLoading } =
+    useMakePost({ networkId, userId, category, processFeedPosting });
+
   return {
-    canPayForPost,
-    makePost,
+    makeSimplePost,
+    makeArticle,
+    makeTrack,
+    makeVideo,
```

Comment:
like we discussed, it's not a good idea to take a category in input and output multiple callbacks, this allows to misuse the function

Reply:
======> I will revert the part "refacto useFeedPosting"